### PR TITLE
MDAKits post

### LINKED ISSUE #241: MDAKits blog post
When we release the initial version of the White Paper https://github.com/MDAnalysis/mdakits-whitepaper/issues/9 we need to publish a blog post to provide some context

- Summarize goal of the MDAKits objective from the CZI EOSS4 grant (and link to post).
- Summarize White Paper.
- Link to DOI for White Paper.
- Introduce the [MDAKits repository](https://github.com/MDAnalysis/MDAKits) and the [MDAnalysis/cookiecutter-mdakit](https://github.com/MDAnalysis/cookiecutter-mdakit).
- Call to action: ask for feedback from the community: use the [MDAKits](https://github.com/MDAnalysis/MDAKits/issues) issue tracker (or any of our other channels).

### PR DESCRIPTION
**Pull request by orbeckst**
fix #241 

Blog post for the release of the white paper. 

* announce white paper
* summarize MDAKits
* ask for feedback

### CODE REVIEW

**Comment by orbeckst** [comment]
There's plenty of work to be done on the post. Please contribute when you can.

File: _posts/2022-08-22-mdakits-intro.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+where you can read the details that go beyond the outline provided in
+this post.
+
+
+## What is an MDAKit?
+
+An MDAKit contains code based on MDAnalysis that solves a specific
+scientific problems or in some form enhances the MDAnalysis core
+library. An MDAKit can be written by anyone and hosted
+anywhere.
```
Comment (IAlibay) [inline _posts/2022-08-22-mdakits-intro.md:23]:
```suggestion
An MDAKit is a package containing code based on MDAnalysis that solves a specific
scientific problems or in some form enhances the MDAnalysis core
library. An MDAKit can be written by anyone and hosted
anywhere.
```

File: _posts/2022-08-22-mdakits-intro.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+<small>(The MDAKit cookiecutter is based off the [Cookiecutter for
+Computational Molecular Sciences (CMS) Python
+Packages](https://github.com/MolSSI/cookiecutter-cms) by Levi N. Naden
+and Jessica A. Nash from the [Molecular Sciences Software Institute
+(MolSSI)](http://molssi.org/) and Daniel G. A. Smith of
+[ENTOS](https://www.entos.ai/). Thank you!)</small>
+
+
+### Registering an MDAKit
```
Comment (IAlibay) [inline _posts/2022-08-22-mdakits-intro.md:93]: We shouldn't advertise this yet please. We can't go into production with what we have currently, it'll just cause chaos.

File: _posts/2022-08-22-mdakits-intro.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+If you want to **register your MDAKit** then create a pull request to
+add a meta data entry `metadata.yaml` to [MDAnalysis:
+MDAKits/mdakits/<YOUR MDAKIT NAME>](https://github.com/MDAnalysis/MDAKits/tree/main/mdakits)
+(where you fill also find a template to get you started). Your PR will
+be reviewed for compliance with the requirements (for right now, see the [White
+Paper][10.6084/m9.figshare.20520726.v1] for specifics). Once
+registered, your MDAKit will be continuously tested.
```
Comment (IAlibay) [inline _posts/2022-08-22-mdakits-intro.md:101]:
```suggestion
Adding packages to the **MDAKit registry** will happen by opening a pull request against the [mdakits repository](https://github.com/MDAnalysis/MDAKits/tree/main/mdakits). Briefly, the process will involve adding a small YAML file with details of the MDAKit we are still ironing out the exact details of the registration process, and will open the process for public participation in the near future.
(where you fill also find a template to get you started). Your PR will..... (continue here please)
be reviewed for compliance with the requirements (for right now, see the [White
Paper][10.6084/m9.figshare.20520726.v1] for specifics). Once
registered, your MDAKit will be continuously tested.
```

**Review by IAlibay — COMMENTED** [review]
Just pushing some unfinished comments before my laptop dies...

File: _posts/2022-08-22-mdakits-intro.md (hunk 3 above)
Comment (orbeckst) [reply _posts/2022-08-22-mdakits-intro.md:101]: I’m not going merge this one yet as there is the “please continue here” incompleteness.

File: _posts/2022-08-22-mdakits-intro.md (hunk 2 above)
Comment (orbeckst) [reply _posts/2022-08-22-mdakits-intro.md:93]: Ok, good that you reviewed ;-)

can you complete your changes—you’re the one who knows best what should be here.

File: _posts/2022-08-22-mdakits-intro.md (hunk 3 above)
Comment (orbeckst) [reply _posts/2022-08-22-mdakits-intro.md:101]: @IAlibay could you block the pr with a request changes so that nobody is tempted to quick- merge?

**Review by IAlibay — CHANGES_REQUESTED** [review]
sorry yeah my laptop just ran out of battery so couldn't keep going, will attempt to finish things a bit later on

File: _posts/2022-08-22-mdakits-intro.md (hunk 3 above)
Comment (IAlibay) [reply _posts/2022-08-22-mdakits-intro.md:101]: updated

File: _posts/2022-08-22-mdakits-intro.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+### Towards publication
+
+The best practices that we encourage MDAKits to fulfill essentially
+amount to the majority of the contribution criteria for submissions to
+software-focused journals such as the [Journal Open Source
+Software](https://joss.theoj.org) (JOSS). We encourage MDAKits to
+consider submission to a journal such as JOSS once they meet the
+required levels of best practices. We are working towards streamlining
+the submission process.
```
Comment (IAlibay) [inline _posts/2022-08-22-mdakits-intro.md:112]:
```suggestion
required levels of best practices. We are working towards 
identifying means of streamlining
the journal submission process.
```

**Review by IAlibay — COMMENTED** [review]
Here are all my text additions. If we want to add Fiona's images then I can look into it later this evening. Otherwise we should aim to merge this as soon as.

**Comment by orbeckst** [comment]
Looking all good to me.

@IAlibay what do you say — you have a "needs changes" review.

@fiona-naughton do you want the cat images go in here? Or do you want a second post where this one is explained properly (with cats)?

**Review by IAlibay — APPROVED** [review]

**Comment by IAlibay** [comment]
@orbeckst we should go ahead with the merge here I think - it's unlikely FIona will be able to do much else before EOD.

**Comment by orbeckst** [comment]
ok!

**Pull request merged**